src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -280,18 +252,95 @@ export default class ComparisonGroupManager extends React.Component<
         }
     }
 
+    @observable private copied = false;
```

Comment:
I'm a little confused, can you explain what's happening with `copied`?

Reply:
Updated the logic. May be its clear now

Reply:
sorry, I still don't get it. what does `copied` represent? maybe a different variable name would make it clearer, e.g. what is copied?

Reply:
forgot to clean it.  it's updated now